src/Tizen.NUI/src/public/RenderEffects/BackgroundBlurEffect.cs

```diff
@@ -0,0 +1,96 @@
+
+
+using System.ComponentModel;
+using System;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// Background blur effect of a View.
+    /// Applications can apply BackgroundBlurEffect as the example below :
+    /// <code>
+    /// BackgroundBlurEffect effect = new BackgroundBlurEffect();
+    /// view.SetRenderEffect(effect);
+    /// </code>
+    /// </summary>
+public class BackgroundBlurEffect : RenderEffect
```

Comment:
```suggestion
    public class BackgroundBlurEffect : RenderEffect
```

Give 4 spacing intent at this class.